components/Footer.vue

```diff
@@ -73,6 +73,15 @@
           <h3 class="link-title h5">Ecosistem Alegeri</h3>
           <nav>
             <ul class="list-unstyled">
+            <li>
+                <a
+                  target="_blank"
+                  rel="noopener noreferrer"
+                  href="https://votdiaspora.ro/"
```

Comment:
@johnsmithm It is ok that we link same page in footer?
```suggestion
                  href="https://votdiaspora.md/"
```

Reply:
not really, we can put the link to this repo https://github.com/code4moldova/vot-diaspora-client